deis-workflow-manager/manifests/deis-workflow-manager-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  type: LoadBalancer
```

Comment:
I defer to you and others on this, @bacongobbler and I set this up as a working example, if a routable application is preferable, let's do that.